.github/workflows/generate-sdk.yaml

```diff
@@ -58,6 +58,21 @@ jobs:
           cp "../generator/package/expediagroup/sdk/$namespace/model.py" "./temp/expediagroup/sdk/$namespace/model.py"
 
           python3 ./__main__.py -p ./temp -n "$namespace"
+      - name: Install Pandoc
+        run: |
+          pip3 install lastversion
+
+          architecture="$(dpkg --print-architecture)"
+          lastversion --assets --filter "$architecture.deb" download https://github.com/jgm/pandoc
+
+          sudo dpkg -i *.deb
```

Comment:
Let's be more specific in what we need to install here, as it's a `sudo` command.